packages/hash/src/xxhash-wasm/get-xxhash-wasm-instance.ts

```
@@ -0,0 +1,16 @@
+import type { XXHashAPI } from 'xxhash-wasm';
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import-x/namespace)